src/App/App.js

```diff
@@ -2,39 +2,76 @@ import React, {Component} from 'react';
 import Header from '../Header/Header';
 import MovieContainer from '../MovieContainer/MovieContainer';
 import MovieDetails from '../MovieDetails/MovieDetails';
-import movieData from '../data.js';
+// import movieData from '../data.js';
```

Comment:
We need to delete this

Reply:
Indeed, commented out code left behind is just sloppy.